backends/advanced/src/advanced_omi_backend/routers/modules/obsidian_routes.py

```
@@ -0,0 +1,227 @@
+        raise
+    except Exception as e:
+        logger.error(f"Failed to process uploaded zip: {e}")
+        raise HTTPException(status_code=500, detail=f"Failed to process uploaded zip: {e}")
+    finally:
+        # Ensure file handle is closed (following upload_files_async pattern)
+        if zip_file_handle:
+            try:
+                zip_file_handle.close()
+            except Exception as close_error:
+                logger.warning(f"Failed to close zip file handle: {close_error}")
+                raise close_error
```

Ruff (0.14.8)

Use `raise` without specifying exception name

Remove exception name

(TRY201)